packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/index.mjml

```diff
@@ -23,15 +23,5 @@
     <mj-text css-class="text-body align-left">
       <span data-l10n-id="passwordChangeRequired-different-password"><b>Important:</b> Pick a different password than what you were previously using and make sure that it is different from your email account.</span>
     </mj-text>
-
```

Comment:
Nice, clean up.

Also, just double checking that this should have been removed. Seems unnecessary. I think I saw this in a thread somewhere.

Reply:
Correct! Confirmed with UX :)